app/uk/gov/hmrc/merchandiseinbaggagefrontend/service/CurrencyConversionService.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.merchandiseinbaggagefrontend.service
+
+import java.time.LocalDate
+
+import uk.gov.hmrc.http.{HeaderCarrier, HttpClient}
+import uk.gov.hmrc.merchandiseinbaggagefrontend.config.CurrencyConversionConf
+import uk.gov.hmrc.merchandiseinbaggagefrontend.model.currencyconversion.CurrencyPeriod
+import uk.gov.hmrc.http.HttpReads.Implicits._
+
+import scala.concurrent.{ExecutionContext, Future}
+
+trait CurrencyConversionService extends CurrencyConversionConf {
```

Comment:
Do we want to follow the pattern (inheritance over composition)?

This looks like a Connector rather than a Service?

Reply:
I followed what was there with regards to the PaymentService implementation as didn't want to mix styles. I'm happy with either

Reply:
Fair enough about being consistent but I think we should refactor them both to be connectors as that's the HMRC standard naming for one of these things.